lib/linguist/languages.yml

```diff
@@ -5134,6 +5134,16 @@ Sieve:
   codemirror_mode: sieve
   codemirror_mime_type: application/sieve
   language_id: 208976687
+Singularity:
+  type: programming
+  color: "#64E6AD"
+  tm_scope: source.singularity
+  extensions:
+  - ".def"
+  filenames:
+  - Singularity
```

Comment:
For other reviewers: it looks like there are a few Bash files with that name, but I'm not overly worried since our shebang strategy should detect them.